### PR TITLE
19050 Fixed future effective check

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#19050

*Description of changes:*
- app version = 5.6.34
- fixed check for future effective filing
- added missing filing statuses to enum

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
Before:

image

After:

image

File: src/components/Amalgamation/AmalgamatingBusinesses.vue [hunk 1]
```diff
@@ -366,6 +366,12 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
       return
     }
 
+    const isFutureEffective = (
+      business.firstFiling.isFutureEffective === true &&
+      business.firstFiling.status !== FilingStatus.COMPLETED &&
```
Comment (JazzarKarim) [inline src/components/Amalgamation/AmalgamatingBusinesses.vue:371]: I see. Nice!

**Review by JazzarKarim — APPROVED** [review]
LGTM!

File: src/components/Amalgamation/AmalgamatingBusinesses.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/Amalgamation/AmalgamatingBusinesses.vue:371]: I should have mentioned this before... this is the same check as here:
https://github.com/bcgov/business-filings-ui/blob/4f6698f3f4fb6e95fef6c4695ac3abb31b2afb21/src/components/Dashboard/FilingHistoryList/HeaderActions.vue#L123

**Pull request merged**